data/ibilalkayy.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "Bilal Khan",
   "type": "personal",
-  "bio": "College Dropout ➞ Self-Taught DevOps Guy | Open Source Contributor to Kubernetes, Content Creator on YouTube & a Technical Writer at Hashnode.",
+  "bio": "College Dropout ➞ Self-Taught DevOps Guy | AWS Community Builder, Open Source Contributor, Content Creator on YouTube & Hashnode.",
 
-  "tags": ["Open Source", "DevOps", "Golang", "Kubernetes", "Docker", "DevRel"],
+  "tags": ["Open Source", "DevOps", "Golang", "DevRel"],
```

Comment:
```suggestion
  "tags": ["OpenSource", "DevOps", "Golang", "DevRel"],
```
📋 tags usually have to _not_ contain spaces. Not sure if EddieHub ™️ supports tags with spaces, I'd have to paw around the code. However, it's a good practice in general when you see "tag", to just skip the spaces. Consistency in that way makes it a lizard brain task. 🤙 

Also, if I said "containerize this npm module / Rust thing" would you know what that means / how to do it? If so, the 2023 version of Docker is `OCIContainers`. Toss that in there and you'll have yacht-money in no-time.

Reply:
I think it supports :)

![image](https://user-images.githubusercontent.com/88102392/221430815-73a4e120-5885-46df-b732-484702eb40a9.png)